pkg/config/config.go

```diff
@@ -371,5 +381,19 @@ func LoadReader(fd io.Reader) (config Project, err error) {
 	}
 	err = yaml.UnmarshalStrict(data, &config)
 	log.WithField("config", config).Debug("loaded config file")
+	if err != nil {
+		return config, err
+	}
+	err = validateConfig(config)
 	return config, err
 }
+
+// validateConfig validates the config if the combination of properties
+// is valid
+func validateConfig(config Project) error {
+	if config.Release.GitHub.String() != "" && config.Release.GitLab.String() != "" {
```

Comment:
maybe do this on `release` pipe on `Defaults` method?

Reply:
fine for me as well. Done in https://github.com/goreleaser/goreleaser/pull/1038/commits/554a020471a84b964aed371cacdf0333b24ca91a